src/platform/SIMULATOR/sitl.c

```diff
@@ -707,22 +709,17 @@ void FLASH_Lock(void)
     }
 }
 
-FLASH_Status FLASH_ErasePage(uintptr_t Page_Address)
+configStreamerResult_e configWriteWord(uintptr_t address, config_streamer_buffer_align_type_t value)
 {
-    UNUSED(Page_Address);
-//    printf("[FLASH_ErasePage]%x\n", Page_Address);
-    return FLASH_COMPLETE;
-}
+    STATIC_ASSERT(CONFIG_STREAMER_BUFFER_SIZE == sizeof(uint32_t), "CONFIG_STREAMER_BUFFER_SIZE does not match written size");
```

Comment:
This can be easily changed to use `config_streamer_buffer_align_type_t` (after renaming, see above).